src/Commands/CheckLicenses.php

```diff
@@ -95,7 +101,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $format = OutputFormat::tryFromInput($formatOption);
 
         $formatter = OutputFormatterFactory::create($format, $io, $this->tableRenderer);
-        $formatter->format($dependencyChecks);
+        $result = $formatter->format($dependencyChecks);
+       
+        /** @var string|null $path */
+        $path = $input->getOption('output');
+        if (is_string($path) && $path !== '') {
+            file_put_contents($path, $result);
```

Comment:
Do we need some validation on this? e.g. is it in a folder that exists? Or do we just wanna expect a single filename and not a path? In which case we might want to prevent having a directory separator in it?